checks.d/network.py

```diff
@@ -176,6 +176,8 @@ def _check_linux(self, instance):
                 metrics = self._parse_linux_cx_state(lines[2:], self.TCP_STATES['netstat'], 5)
                 for metric, value in metrics.iteritems():
                     self.gauge(metric, value)
+            except Exception:
+                self.log.exception("Error collecting connection stats.")
```

Comment:
yeah, it's not necessary, I just want to make sure we aren't silencing errors we don't expect. Best practice is to attempt to keep from running `except Exception:` as much as possible, for that reason.

Could we do a `catch SubprocessOutputEmptyError:` here instead?